en/docs/quick-starts/webapp-custom-sample.md

```diff
@@ -24,28 +24,31 @@ Please refer ['Write a Custom Inbound Protocol'](insertlink) section for the imp
 ---
 ## Deploy the inbound authenticator
 
- - Copy the TODO-authenticator.jar to the `<IS_HOME>/repository/components/dropins` directory.
+ - Copy the [authenticator.jar](TODO:insert-download-link) to the `<IS_HOME>/repository/components/dropins` directory.
  
 ----
 
+(TODO: dev-portal-fragment)
+
 {!fragments/register-a-service-provider.md!}
 
-4. TODO :Expand **Inbound Authentication Configuration** and fill the relevant data in the **Custom Inbound Configuration section.
+4. Expand **Inbound Authentication Configuration** and fill the relevant data in the **Custom Inbound Configuration section.
 ![inbound authentication configuration](../assets/img/guides/custom-inbound-authenticator-config.png)
 
 ----
 
 ## Deploy and test the sample web app
-1. Copy the extracted and modified TODO folder to the `<TOMCAT_HOME>/webapps` folder
+1. Copy the extracted and modified (TODO) folder to the `<TOMCAT_HOME>/webapps` folder
 
 2. Access the TestApp application at [http://localhost:8080/TestApp](insertlink)
 
 3. Click **Login**
 
-3. TODO: Explain the behaviour based on the sample.
+3. (TODO: Explain the behaviour based on the sample)
 
 
+----
 
 !!! info "Related Topics"
- - [Enable Login for a Web Application](../../../guides/login/integrate-custom-protocol)
- - [Write a Custom Inbound Protocol](inserlink)
\ No newline at end of file
+    - [Guide: Enable Login for a Web Application](../../../guides/login/integrate-custom-protocol)
+    - [Guide: Write a Custom Inbound Protocol](TODO:insert-link)
```

Comment:
Why doesn't the placeholder mention if it is a guide or concept?

Reply:
since the link itself mentions guide and its clear what needs to be added there, thought its okay

Reply:
If we search for this based on the default place holders we decided on, this might be missed.